### ISSUE TITLE
Can the Audio Plugin be configured to use another output device?

### ISSUE THREAD

**Issue by Shaftoe62** [SHWotever/SimHub#846]
I use ShakeIt quite extensively for 6 bass shakers but I also use the game sound (engine sound) for my Buttkicker. Mixed with ShakeIt sound through Voicemeeter.

Recently I've started to add sounds with Dash Studio and the Audio Plugin. But those sounds are send to the active windows sound device. So they get mixed with the game sound which is (also) send to the Buttkicker.

That's not what I want. I want the sounds I add with Dash Studio to be exclusively send to my audio system and/or headset.

To do that I need to be able to select another output device for the Simhub Audio Plugin.

Which I dont't think I can?

**Comment by SHWotever** [comment]
Hi ! No it's not possible yet, currently it's simply sent only to the default windows sound device indeed, 
You can watch this feature request to know when/if it gets implemented : https://github.com/SHWotever/SimHub/issues/661

Nicolas

**Comment by Shaftoe62** [comment]
Ok I subscribed to that thread.

But I also found a kind of workaround. The purpose was/is to be able to play music from specific countries to enhance the atmosphere in Dirt Rally (Without shaking out of my chair with a powerful beat :-).

I now use the Windows Media Player to do that. I can configure WMP to use a specific output device and I can control it a little bit from within SimHub.

It would be nice to have some more control over WMP but I'll put that in another "Issue".
